azavea/pwd-fishway
# Create windows install workflow

## Overview

Use this pipeline to install the app to the Windows computer. For app updates we'll probably send the zip file to the museum tech consultants, Greenhouse, but may also have to run over to the museum to do an install.

Connects #76

## Testing Instructions

Run `./scripts/cipublish` locally from within the VM and see the project zips up all its built assets and the app runs correctly from the `index.html` file. 

Currently a stale version is hosted on the Windows computer from `localhost` root and it'll be updated when we're ready to ship! 🚢🚀
